GUI/MainInstall.cs

```diff
@@ -96,6 +97,31 @@ public partial class Main
                 installCanceled = true;
             };
 
+            GUI.user.RaiseMessage("About to install...\r\n");
```

Comment:
Whoops, I deleted this comment from the commit in order to move it to the PR:
"This message is appearing when the only actions are removing mods, and the client seems to get stuck after that."